src/server/api/admin_api.py

```diff
@@ -61,15 +68,32 @@ def execute():
     statistics = get_statistics()
 
     last_execution_details = {"executionTime": current_time, "stats": statistics}
+    last_ex_json = (json.dumps(last_execution_details))
 
-    last_execution_file = open(LOGS_PATH + "last_execution.json", "w")
-    last_execution_file.write(json.dumps(last_execution_details))
-    last_execution_file.close()
+    # Write Last Execution stats to DB  
+    # See Alembic Revision ID: 05e0693f8cbb for table definition
+    with engine.connect() as connection:
+        ins_stmt = insert(kvt).values(               # Postgres-specific insert() supporting ON CONFLICT 
+            keycol = 'last_execution_time',
+            valcol = last_ex_json,
+            )
+        # If key already present in DB, do update instead 
+        upsert = ins_stmt.on_conflict_do_update(
```

Comment:
don't we prefer just storing a new row? so we keep a history? we can add a timestamp to each row we add, and the UI will always pull the latest.

Reply:
We could - what would we do with that history?